selector.go

```diff
@@ -0,0 +1,28 @@
+package datadog
+
+import (
+	"go.opentelemetry.io/otel/api/metric"
+	export "go.opentelemetry.io/otel/sdk/export/metric"
+	"go.opentelemetry.io/otel/sdk/metric/aggregator/array"
+	"go.opentelemetry.io/otel/sdk/metric/aggregator/lastvalue"
+	"go.opentelemetry.io/otel/sdk/metric/aggregator/sum"
+)
+
+type (
+	selectorDataDog struct{}
+)
+
+func NewWithDataDogMeasure() export.AggregationSelector {
```

Comment:
Since this wrapper is mainly intended for Observer callbacks which are not related to Measures, I think we can have a better name (with some documentation as well, because hopefully we won't need this in the near future). Something like this: 

```golang

// NewAggregatorWrapper ensures that all ObserverKinds are aggregated as LastValue metrics 
// so that this exporter can export them as Gauge metric kinds. This is necessary until https://github.com/open-telemetry/oteps/pull/88 is resolved and implemented. 
func NewAggregatorWrapper